lib/src/components/NxSegmentedButton/NxSegmentedButton.scss

```diff
@@ -69,8 +69,8 @@
       border-bottom-style: none;
       border-left-style: none;
 
-      padding-top: calc(var(--nx-spacing-2x) + 1px);
-      padding-bottom: calc(var(--nx-spacing-2x) + 1px);
+      padding-top: var(--nx-spacing-2x);
+      padding-bottom: var(--nx-spacing-2x);
       padding-left: calc(var(--nx-spacing-6x) + 1px);
```

Comment:
I suspect this and the `+1` on the other half's `padding-right` also aren't supposed to be here anymore.  It would be weird to include the border in the spacing on one axis but not the other.

Reply:
> I suspect this and the `+1` on the other half's `padding-right` also aren't supposed to be here anymore. It would be weird to include the border in the spacing on one axis but not the other.

Fixed here: ff880780732be0bf892fef562bbd689c617c28d9